rats-tls/README.md

```diff
@@ -51,11 +51,40 @@ make -C build install
 ```
 
 If you want to run TDX instances, please type the following command.
+
 ```shell
 cmake -DRATS_TLS_BUILD_MODE="tdx" -DBUILD_SAMPLES=on -H. -Bbuild
 make -C build install
 ```
 
+3. Make package for RATS TLS 
+
+```shell
+cd inclavare-containers/rats-tls
+```
+
+If you want to build host mode related package, please type the following command.
+
+```shell
+make package MODE=host
+```
+
+If you want to build occlum mode related package, please type the following command.
+```shell
+make package MODE=occlum
+```
+
+If you want to build sgx mode related package, please type the following command.
+```shell
+make package MODE=sgx
+```
+
+If you want to build tdx mode related package, please type the following command.
+
+```shell
+make package MODE=tdx
+```
+
```

Comment:
Thank you. 
I'm not sure if it's necessary to make the `make package` command support different modes.  
I think developers can call the `make package` command to generate packages in four modes: host, occlum, tdx, and sgx, and then maintain these packages to the IC repository. 
Users can install different packages according to their needs.

Reply:
Do you mean the different package for each .so or .a?  The finally style will be that format but i don't think the style will be in such stage. Currently stage, 
In future mode, rats-tls after redesigned arch will be divided into three levels: core level/transport level/lib rats level, then we will make the install package according to different levels will make sense.